image-index.md

```diff
@@ -157,14 +156,9 @@ When the variant of the CPU is not listed in the table, values are implementatio
       }
     },
     {
-      "mediaType": "application/vnd.oci.artifact.manifest.v1+json",
+      "mediaType": "application/vnd.oci.image.index.v1+json",
```

Comment:
Was this intended to be image manifest, not index? Nested indexes were rejected as the intended impl. in proposal F

Reply:
A nested index is explicitly supported by an image index: https://github.com/opencontainers/image-spec/blame/188d3a4e194241cf3412010a6c1073a71d4b0708/image-index.md#L42

This section is "Example showing an image index pointing to manifests with multiple media types", so I changed this to an index, which happens to be difference from the image media type.

Reply:
You can mark this resolved.